### PR TITLE
Updates to Azure Batch Client & Task

### PR DESCRIPTION
**Pull request by praneet22**
## Description

- Added method to print batch exceptions
- Added logic to run simple command on one node
- Updated the method parameter passing using self

### CODE REVIEW

File: luigi/contrib/azurebatch.py [hunk 1]
```diff
@@ -415,42 +441,58 @@ class AzureBatchTask(luigi.Task):
...
-    output_path = luigi.Parameter()
+    input_path = luigi.Parameter(default=" ")
+    command = luigi.Parameter(default="echo Hello World")
+    output_path = luigi.Parameter(default=" ")
     pool_id = luigi.Parameter(default=TASK_POOL_ID)
     job_id = luigi.Parameter(default=TASK_JOB_ID)
     pool_node_count = luigi.IntParameter(default=TASK_POOL_NODE_COUNT)
     pool_vm_size = luigi.Parameter(default=TASK_POOL_VM_SIZE)
     kwargs = luigi.DictParameter(default={})
+    container_name = "luigitargetdata"
```
Comment (awaemmanuel) [inline luigi/contrib/azurebatch.py]: This should also be a `luigi.Parameter(default="luigiTargetData")`

File: luigi/contrib/azurebatch.py [hunk 2]
```diff
@@ -415,42 +441,58 @@ class AzureBatchTask(luigi.Task):
...
+            self.batch_account_key,
+            self.batch_account_url,
+            self.storage_account_name,
+            self.storage_account_key,
+            self.output_path,
+            self.pool_id,
+            self.job_id,
+            self.pool_node_count,
+            self.pool_vm_size,
+            # self.kwargs,
```
Comment (awaemmanuel) [inline luigi/contrib/azurebatch.py:472]: Clean this up.

**Review by awaemmanuel — CHANGES_REQUESTED** [review]

File: luigi/contrib/azurebatch.py (hunk 1 above)
Comment (praneet22) [reply luigi/contrib/azurebatch.py]: We are creating this as tmp container & deleting it after, thus though of fixing it. However, yes I will make it a luigi parameter with default

Comment (praneet22) [reply luigi/contrib/azurebatch.py]: Done

File: luigi/contrib/azurebatch.py (hunk 2 above)
Comment (praneet22) [reply luigi/contrib/azurebatch.py:472]: This needs to be fixed. I have commented it as we do not need it now

Comment (awaemmanuel) [reply luigi/contrib/azurebatch.py:472]: ok

**Review by awaemmanuel — APPROVED** [review]

**Pull request merged**